test/unit/spec/narrative_core/upload/importSetup-spec.js

```diff
@@ -289,30 +286,68 @@ define([
                 });
             });
 
+            /**
+             * stagingResponse is optional. If present, this will start with mocking a call to the
+             * bulk_specification endpoint of the staging area, expecting a 200 response with the
+             * given data.
+             * @param {string} fileName
+             * @param {Object} stagingResponse expected response from a happy staging area call
+             */
+            async function testWithExpectedImportErrors(fileNames, stagingResponse) {
```

Comment:
Broke this out into a little function, as I expect there to be a few more as this bit of the project goes on...